src/app/layout.tsx

```diff
@@ -5,7 +5,6 @@ import type { Metadata } from 'next'
 import './globals.css'
 import '@mantine/core/styles.layer.css'
 import 'mantine-datatable/styles.layer.css'
-import '@mantine/dropzone/styles.layer.css'
```

Comment:
removing this removes the styles for the dropzone stuff, so we still need it included